src/main/java/com/pmm/budgetcalculator/expensing/entity/Expense.java

```diff
@@ -26,8 +23,10 @@ public class Expense extends BaseEntity {
     private String expenseDescription;
     @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
     private Date expenseTime;
+    @NonNull
```

Comment:
NonNull is cool, but what if I send here text? You can validate that as well. Also can I post amount -111111.1111111111111 or should it be only price-like so positive number and 2 decimals?